package.json

```diff
@@ -258,7 +258,7 @@
     "@metamask-institutional/transaction-update": "^0.2.6",
     "@metamask-institutional/types": "^1.2.0",
     "@metamask/abi-utils": "^2.0.2",
-    "@metamask/account-watcher": "^4.1.2",
+    "@metamask/account-watcher": "^4.1.3",
```

Comment:
This version is also required now that the account creation flow no longer use `displayConfirmation: false` for all preinstalled Snaps.